doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,190 @@ Table 7-2 shows security capabilities
+| sec.lcm.010 | The Platform **must** log all changes to geolocation along with the mechanisms and sources of location information (i.e. GPS, IP block, and timing). |  | |
+| sec.lcm.011 | The Platform **must** implement Security life cycle management processes including proactively update and patch all deployed NFVI software. | | |
+
+<a name="7.11.7"></a>
+### 7.11.7. Monitoring and Security Audit
+
+The Platform is assumed to provide configurable alerting and notification capability and the operator is assumed to have automated systems, policies and procedures to act on alerts and notifications in a timely fashion. In the following the monitoring and logging capabilities can trigger alerts and notifications for appropriate action.
+
+| Ref | Requirement | Unit | Definition |
+|---|----|---|----|
+| sec.mon.001 | Platform must provide logs and these logs must be regularly scanned for events of interest |  | |
+| sec.mon.002 | Security logs must be time synchronised |  | |
+| sec.mon.003 | The Platform must log all changes to time server source, time, date and time zones |  | |
+| sec.mon.004 | The Platform must secure and protect Audit logs (contain sensitive information) both in-transit and at rest |  | |
+| sec.mon.005 | The Platform must Monitor and Audit various behaviours of connection and login attempts to detect access attacks and potential access attempts and take corrective actions accordingly |  | |
+| sec.mon.006 | The Platform must Monitor and Audit operations by authorized account access after login to detect malicious operational activity and take corrective actions accordingly |  | |
+| sec.mon.007 | The Platform must Monitor and Audit security parameter configurations for compliance with defined security policies |  | |
+| sec.mon.008 | The Platform must Monitor and Audit externally exposed interfaces for illegal access (attacks) and take corrective security hardening measures |  | |
+| sec.mon.009 | The Platform must Monitor and Audit service handling for various attacks (malformed messages, signalling flooding and replaying, etc.) and take corrective actions accordingly |  | |
+| sec.mon.010 | The Platform must Monitor and Audit running processes to detect unexpected or unauthorized processes and take corrective actions accordingly |  | |
+| sec.mon.011 | The Platform must Monitor and Audit logs from infrastructure elements and workloads to detected anomalies in the system components and take corrective actions accordingly |  | |
+| sec.mon.012 | The Platform must Monitor and Audit Traffic patterns and volumes to detect inbound or outbound DoS attacks, and to prevent malware download attempts |  | |
+| sec.mon.013 | The Security monitoring system must not affect the security (integrity and confidentiality) of the infrastructure, workloads, or the user data (through back door entries). |  | |
```

Comment:
```suggestion
| sec.mon.013 | The monitoring system must not affect the security (integrity and confidentiality) of the infrastructure, workloads, or the user data (through back door entries). |  | |
```
Removing the mention of 'security' monitoring system as that suggests a whole new system, separate from the log monitoring.

Reply:
@kedmison Done -- Thanks